includes/admin/class-wc-rest-stripe-account-keys-controller.php

```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ * Class WC_REST_Stripe_Account_Keys_Controller
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST controller for saving Stripe's test/live account keys. This includes Live Publishable Key,
+ * Live Secret Key, Webhook Secret.
```

Comment:
As per [the "PHP Documentation Standards" article from WP's Coding Standards Handbook](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#2-classes) (which I'm not sure we are following in WC Stripe):

> Class DocBlocks should be formatted as follows:
> 
> * Summary: A brief, one sentence explanation of the purpose of the class spanning a maximum of two lines. Use a period at the end.
> * Description: A supplement to the summary, providing a more detailed description. Use a period at the end.

```suggestion
 * REST controller for saving Stripe's test/live account keys.
 *
 * This includes Live Publishable Key, Live Secret Key, Webhook Secret.
```

Reply:
Updated.